subprojects/docs/build.gradle

```diff
@@ -16,6 +17,11 @@ plugins {
     id 'gradlebuild.split-docs'
 }
 
+class DocsBuildFlags {
```

Comment:
Can we put everything in `https://github.com/gradle/gradle/blob/d758a256cf3c937854d80ddf14e9f6009958ebf3/build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt#L72`? `BuildParams` is supposed to be a container of all parameters.